go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 	sigs.k8s.io/yaml v1.4.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v1.10.0-alpha.0.0.20241016111016-bb93758bd51f
+	volcano.sh/apis v1.10.0-alpha.0.0.20241218081838-e5d361b6bfbe
```

Comment:
There is a new patch in api repo, please user commit id `6ca7b0187107448a5477d08ade59dc31ac3a13a9` to update go mod.

Reply:
done!